src/main/scala/no/ndla/audioapi/controller/AudioController.scala

```diff
@@ -119,13 +120,13 @@ trait AudioController {
 
     get("/", operation(getAudioFiles)) {
       val query = paramOrNone("query")
-      val language = paramOrNone("language")
+      val language = paramOrDefault("language", Language.AllLanguages)
       val license = paramOrNone("license")
       val sort = paramOrNone("sort")
       val pageSize = paramOrNone("page-size").flatMap(ps => Try(ps.toInt).toOption)
       val page = paramOrNone("page").flatMap(idx => Try(idx.toInt).toOption)
 
-      search(query, language, license, sort, pageSize, page)
+      search(query, Some(language), license, sort, pageSize, page)
```

Comment:
Nå begynner det å bli bra! :+1:
Jeg finner en siste bug vi må fikse før denne kan merges:
```
GET /audio-api/v1/audio
{
    "totalCount": 25,
    "page": 1,
    "pageSize": 10,
    "language": "",
    "results": [
        {
            "id": 14,
            "title": "Familien som spela vekk jola ",
            "url": "http://localhost:30014/audio-api/v1/audio/14",
            "license": "by-nc-nd",
            "supportedLanguages": [
                "unknown"
            ]
        }, ...
    ]
}
```

`supportedLanguages` inneholder ikke alle støttede språk for ressursen. I tilegg går det ikke an å slå opp på den `id`n med language == "unknown":
```
GET /audio-api/v1/audio/14?language=unknown
{
    "code": "NOT_FOUND",
    "description": "Audio with id 14 not found",
    "occuredAt": "2017-07-17T08:48:21Z"
}
```